doc/interfaces/tracking_flow.rst

```diff
@@ -0,0 +1,180 @@
+metrics files, stopping_files, and seeding_files. The optional arguments can
+also be provided. The default tracking method is ``eudx``. In this case, we will
+specify the number of seeds per dimension inside voxel (``seed_density``), the
+tracking method (``tracking_method``) and the output directory (``out_dir``).
+
+To get the stopping file, we will create a binary file of GFA of the CSA
+reconstruction method and set a stopping criterion by calling the ``dipy_mask``
+command::
+
+    dipy_mask recons-csa/gfa.nii.gz 0.25 --out_dir "stop_file"
+
+To get the seeding file, we will create the WM mask using median Otsu
+thresholding method by calling the ``dipy_median_otsu`` command on
+``stanford_pve_maps`` dataset::
+
+    dipy_median_otsu data/stanford_hardi/HARDI150.nii.gz pve_wm.nii.gz --out_dir "wm_mask"
+
+There are four tracking methods as described below.
+
+EuDX Tracking
+*************
+
+We will first create the directory in which to save the output tractogram file
+(e.g.: ``eudx_tracking_output``)::
+
+    mkdir eudx_tracking_output
+
+Then, to perform the EuDX tracking we will run the ``dipy_track`` command as::
+
+    dipy_track recons-csa/peaks.pam5 stop_file/mask.nii.gz wm_mask/brain_mask.nii.gz --out_dir "eudx_tracking_output"
+
+Deterministic Tracking
+**********************
+
+Deterministic maximum direction getter is the deterministic version of the 
+probabilistic direction getter. It can be used with the same local models and
+has the same parameters. Deterministic maximum fiber tracking follows the
+trajectory of the most probable pathway within the tracking constraint (e..g.
+max angle). In other words, it follows the direction with the highest
+probability from a distribution, as opposed to the probabilistic direction
+getter which draws the direction from the distribution. Therefore, the maximum
+deterministic direction getter is equivalent to the probabilistic direction
+getter returning always the maximum value of the distribution.
+
+Deterministic maximum fiber tracking is an alternative to EuDX deterministic
+tractography and unlike EuDX does not follow the peaks of the local models but
+uses the entire orientation distributions.
+
+We will first create the directory in which to save the output tractogram file
+(e.g.: ``det_tracking_output``)::
+
+    mkdir det_tracking_output
+
+Then, to perform the deterministic tracking we will run the ``dipy_track``
+command as::
+
+    dipy_track recons-csa/peaks.pam5 stop_file/mask.nii.gz wm_mask/brain_mask.nii.gz --seed_density 2 --tracking_method "det" --out_dir "det_tracking_output"
+
+Probabilistic Tracking
+**********************
+
+Probabilistic fiber tracking is a way of reconstructing white matter connections
+using diffusion MR imaging. Like deterministic fiber tracking, the probabilistic
+approach follows the trajectory of a possible pathway step by step starting at a
+seed, however, unlike deterministic tracking, the tracking direction at each
```

Comment:
(...) seed: however, unlike deterministic tracking, (...)

Reply:
@jhlegarreta do you mean that 'starting at a seed' should be removed?

Reply:
I had mistyped my suggestion, sorry: I've just edited it. Just change the comma after "seed" by a semi-colon. The text is OK as it is.

Reply:
Addressed in d929e25